### PR TITLE
Update cordova and cordova android to latest versions

### PR DESCRIPTION
**Pull request by vitorsemeano**
This changes are based on the changes made by @dkanada. I just updated to latest versions of those packages, used latest upstream version, and also fixed doclker-build.sh so it could point to the right path of the built apk.

Only thing i didn't test was the generated apk by the docker build process.

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <icon src="res/icon.png" />
     <platform name="android">
         <preference name="android-minSdkVersion" value="19" />
-        <preference name="android-targetSdkVersion" value="27" />
+        <preference name="android-targetSdkVersion" value="28" />
```
Comment (JustAMan) [inline config.xml]: why?

File: config.xml [hunk 2]
```diff
@@ -29,8 +29,8 @@
         <icon density="xxhdpi" src="res/android/xxhdpi/icon.png" />
         <icon density="xxxhdpi" src="res/android/xxxhdpi/icon.png" />
     </platform>
-    <engine name="android" spec="^6.3.0" />
     <plugin name="cordova-plugin-whitelist" spec="^1.3.3" />
     <plugin name="acidhax.cordova.chromecast" spec="https://github.com/jellyfin/cordova-plugin-chromecast" />
     <plugin name="org.jellyfin.mobile" spec="./src/NativeShell" />
+    <engine name="android" spec="^8.0.0" />
```
Comment (JustAMan) [inline config.xml]: does this mean Android older than Oreo is doomed?

File: config.xml (hunk 1 above)
Comment (vitorsemeano) [reply config.xml]: Cordova-android in the version 8.0.0 introduces support for Android API level 28. Si think we should take advantage of new APIs for more recent devices that use this app. Android devices with at least API level 19 are still supported.

File: config.xml (hunk 2 above)
Comment (vitorsemeano) [reply config.xml]: No, thats only a indicator for the cordova-android version being used. If you go to package.json you will see that the package cordova-android is targeted to at least 8.0.0

File: config.xml (hunk 1 above)
Comment (vitorsemeano) [reply config.xml]: But if you think it's not necessary or I said something wrong, I think there's no problem reverting this change.

Comment (JustAMan) [reply config.xml]: I didn't see anything about "minimal version", if it's still 19 no problem.
I just wanted to make sure we don't drop support of not-very-fresh phones.

**Pull request merged**